lib/middleware.ts

```diff
@@ -22,7 +22,7 @@ export default function middleware(config: Line.Config & Line.MiddlewareConfig):
     const signature = req.headers["x-line-signature"] as string;
 
     if (!signature) {
-      next();
+      next(new SignatureValidationFailed("no signature", signature));
```

Comment:
`signature` here may be `undefined`, so it is not really meaningful to pass it. How about making `public signature` in `SignatureValidationFailed` optional and just not passing it?

- https://github.com/line/line-bot-sdk-nodejs/blob/master/lib/exceptions.ts#L4

``` typescript
constructor(
  message: string,
  public signature?: string,
) {
```